pulseapi/users/serializers.py

```diff
@@ -1,7 +1,24 @@
 """Serialize the models"""
 from rest_framework import serializers
 
-from pulseapi.users.models import EmailUser
+from pulseapi.users.models import (
+    EmailUser,
+    UserFavorites,
+)
+
+
+class UserFavoritesSerializer(serializers.ModelSerializer):
+    """
+    Serializes a {user,entry,when} fave.
+    """
+
+    user = '__all__'
```

Comment:
`__all__` normally exposes all public methods for use in `import ... from ... ` statements including those prefixed with underscores (usually 'hidden') but in this case it doesn't look like it actually does anything, so I'll remove it.